README.md

```diff
@@ -29,6 +29,13 @@ Invoke the test runner from the root directory as follows:
 
     tox
 
+To feed a sample file via ``curl``, start the container as mentioned above and call:
+
+    curl -H "Authorization: Bearer admin" \
```

Comment:
```suggestion
    curl \
```
the bearer token was leftover from my curl test against the mcode repo, shouldn't be necessary here

Reply:
i sorta like the placeholder... but as you wish!